src/main/java/Command.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Represents an executable command
+ */
+public abstract class Command {
+    TaskList taskList;
+
+    /**

```

Comment:
I think it's really good that you implemented Command as an abstract class! Makes the code and implementation neater and easier to follow. I did that too.